### DISCUSSION TITLE
navbar jumps to different external links depending on the language

### DISCUSSION THREAD

**Discussion by casionone** [facebook/docusaurus#8930] [General]
how to config navbar jumps to different external links depending on the language
forexample：
click home button jump to https://www.google.com/
![image](https://user-images.githubusercontent.com/7869972/234509319-a94df5f4-51de-40c1-b0f2-3ff90dd3e4f2.png)
click home button jump to https://www.facebook.com after configuration zh language
![image](https://user-images.githubusercontent.com/7869972/234509574-b0e44aee-883c-408c-b7cd-5d77b62598e8.png)

**Comment by slorber** [comment]
We don't have a good API to do that, but there's a workaround you can use thanks to the `process.env. DOCUSAURUS_CURRENT_LOCALE` available in the config file, see https://github.com/facebook/docusaurus/pull/8677
